i3df/build.rs

```diff
@@ -81,7 +84,7 @@ fn create_dtype(
             "f32" => quote! {f32},
             t => quote! { ERROR_NOT_IMPLEMENTED: #t },
```

Comment:
Maybe nitpicking in existing code here, but would it make sense to use [`compile_error!`](https://doc.rust-lang.org/std/macro.compile_error.html) here for a better error message?
```suggestion
             t => {
               let message = format!("Type not implemented: {}", t);
               quote!{ compile_error!(#message) }
             },
```